keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/optimizer/Optimizer.java

```diff
@@ -90,8 +93,11 @@ static Optimizer ofConnectedGraph(Vertex<?> vertexFromNetwork) {
         return of(vertexFromNetwork.getConnectedGraph());
     }
 
-    static double[] currentPoint(List<? extends Vertex<? extends NumberTensor>> continuousLatentVertices) {
-        long totalLatentDimensions = totalNumberOfLatentDimensions(continuousLatentVertices);
+    static double[] convertToPoint(List<String> latentVariables,
```

Comment:
you don't need the first argument, `latentVariables`, because you have the same information in the keys of the 2nd and 3rd arguments

Reply:
👍

Reply:
Actually, the list is needed because the order needs to be preserved.